src/wallet/rpcwallet.cpp

```diff
@@ -1042,7 +1043,7 @@ UniValue sendmany(const UniValue& params, bool fHelp)
             "      \"address\":amount   (numeric) The Zcash address is the key, the numeric amount in " + CURRENCY_UNIT + " is the value\n"
             "      ,...\n"
             "    }\n"
-            "3. minconf                 (numeric, optional, default=1) Only use the balance confirmed at least this many times.\n"
+            "3. minconf                 (numeric, optional, default=10) Only use the balance confirmed at least this many times.\n"
```

Comment:
Well, the default is actually whatever is specified by `orchardanchorconfirmations`, which itself defaults to 10. So really we should be transplanting whatever the value of `nOrchardAnchorConfirmations` currently is here into this string?